src/java/picard/analysis/CollectRawWgsMetrics.java

```diff
@@ -5,18 +5,43 @@
 import picard.cmdline.programgroups.Metrics;
 
 /**
- * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing experiments, same implementation as CollectWgsMetrics, with different defaults: lacks baseQ and mappingQ filters and has much higher coverage cap.
+ * Computes a number of metrics that are useful for evaluating coverage and performance of whole genome sequencing
+ * experiments, same implementation as CollectWgsMetrics, with different defaults: lacks baseQ and mappingQ filters
+ * and has much higher coverage cap.
  *
  * @author farjoun
  */
 @CommandLineProgramProperties(
-        usage = "Computes a number of metrics that are useful for evaluating coverage and performance of " +
-                "whole genome sequencing experiments. Defaults are different than for CollectWgsMetrics.",
-        usageShort = "Writes whole genome sequencing-related metrics for a SAM or BAM file",
+        usage = CollectRawWgsMetrics.USAGE_SUMMARY + CollectRawWgsMetrics.USAGE_DETAILS,
+        usageShort = CollectRawWgsMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectRawWgsMetrics extends CollectWgsMetrics{
-
+    static final String USAGE_SUMMARY = "Writes whole genome sequencing-related metrics for a SAM or BAM file.  ";
+    static final String USAGE_DETAILS = "Useful metrics for evaluating coverage and performance of whole " +
+            "genome sequencing experiments.  Although similar to the CollectWgsMetrics tool," +
+            " the default thresholds for CollectRawWgsMetrics are less stringent.  For example, the CollectRawWgsMetrics" +
+            " has lower base and mapping quality score thresholds as well a higher coverage cap than the CollectWgsMetrics" +
+            " tool.<br /><br />" +
+            "Histogram output is optional and for a given run, displays two separate outputs on the y-axis while using a single set" +
+            " of values for the x-axis.  Specifically, the first column in the histogram table (x-axis) is labeled \"coverage\" and represents" +
+            " different possible coverage depths.  However, it also represents the range of values for the base quality scores and thus should probably be" +
+            " labeled \"sequence depth and base quality scores\".  " +
+            "The second and third columns (y-axes) correspond to the numbers of bases at a specific sequence depth" +
+            " \"count\" and the numbers of bases at a particular base quality score \"baseq_count\" respectively." +
+            "<br />" +
+            "<h4>Usage example:</h4>" +
+            "<pre>" +
+            "java -jar picard.jar CollectRawWgsMetrics \\<br />" +
+            "      I=myBAM.bam \\<br />" +
+            "      O=rawwgsmetrics.txt \\<br />" +
+            "      R=referenceSeq.fasta \\<br />" +
+            "      INCLUDE_BQ_HISTOGRAM=true" +
+            "</pre>" +
```

Comment:
Can we review and standardize the file names we use in all these tools?  Elsewhere I've seen "input.bam" and "reference_sequence.fasta" (as opposed to myBAM.bam and referenceSeq.fasta here), and I think those are preferable.